tests/perl.yaml

```diff
@@ -0,0 +1,20 @@
+#-------------------------------------------------------------------
```

Comment:
Instead of creating a new test suite file just for perl (we're only ever going to have this one single hello world test). Let's just add this test to the `tests/app.yml` file.